### PR TITLE
New Billing FAQ Page (Usage Metrics)

### PR DESCRIPTION
**Pull request by wissallahjouji**
Usage Metrics is now GA (!!!) This is the public FAQ for the new feature.

### CODE REVIEW

File: content/en/account_management/billing/usage_metrics.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Overview
+
+Usage metrics are estimates of your current Datadog usage in near real-time. They enable you to:
+
+* Graph your estimated usage 
+* Create monitors around your estimated usage based on thresholds of your choosing
+* Get instant alerts of spikes or drops in your usage
+* Assess the potential impact of code changes on your usage in near real-time
+
+**Note:** These usage metrics are estimates that won't always match up to billable usage given their real-time nature. There is a 10-20% difference between estimated usage and billable usage on average.
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/usage_metrics.md]:
```suggestion
**Note**: These usage metrics are estimates that won't always match up to billable usage given their real-time nature. There is a 10-20% difference between estimated usage and billable usage on average.  
```

File: content/en/account_management/billing/usage_metrics.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* Graph your estimated usage 
+* Create monitors around your estimated usage based on thresholds of your choosing
+* Get instant alerts of spikes or drops in your usage
+* Assess the potential impact of code changes on your usage in near real-time
+
+**Note:** These usage metrics are estimates that won't always match up to billable usage given their real-time nature. There is a 10-20% difference between estimated usage and billable usage on average.  
+
+{{< img src="account_management/billing/usage-metrics-01.png" alt="Dashboard Example" responsive="true">}}
+
+### Types of Usage
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/usage_metrics.md]:
```suggestion
### Types of usage
```

File: content/en/account_management/billing/usage_metrics.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Estimated usage metrics are generally available for the following usage types: 
+
+| Usage Type         | Metric                                                                                              |
+|--------------------|-----------------------------------------------------------------------------------------------------|
+| Infra. Hosts       | `datadog.estimated_usage.hosts`                                                                     |
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/usage_metrics.md]:
```suggestion
| Infrastructure Hosts       | `datadog.estimated_usage.hosts`                                                                     |
```

File: content/en/account_management/billing/usage_metrics.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+| Containers         | `datadog.estimated_usage.containers`                                                                |
+| Custom Metrics     | `datadog.estimated_usage.metrics.custom`                                                            |
+
+{{< img src="account_management/billing/usage-metrics-02.png" alt="Metric Names" responsive="true">}}
+
+More usage types will be added in the near future.
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/usage_metrics.md]:
```suggestion
```
We don't talk about the future in the docs

File: content/en/account_management/billing/usage_metrics.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+### Multi-Org Usage
```
Comment (ruthnaebeck) [inline content/en/account_management/billing/usage_metrics.md]:
```suggestion
### Multi-Org usage
```

File: content/en/account_management/billing/usage_metrics.md (hunk 4 above)
Comment (wissallahjouji) [reply content/en/account_management/billing/usage_metrics.md]: Makes sense.

**Pull request merged**